registry/storage/driver/middleware/cloudfront/s3filter.go

```diff
@@ -0,0 +1,166 @@
+package middleware
```

Comment:
Name the package for the functionality.

Reply:
This was already the name of the package, rename should be done in a different PR